livekit-ffi/src/proto.rs

```diff
@@ -1,4 +1,32 @@
 #![allow(non_snake_case)]
 #![allow(clippy::enum_variant_names)]
 
+use livekit::ChatMessage as RoomChatMessage;
+
 include!("livekit.proto.rs");
+
+impl From<ChatMessage> for RoomChatMessage {
```

Comment:
currently defining these conversions both in livekit-ffi and in livekit, which feels wrong, but couldn't make it work otherwise.